src/pages/community/meetups.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Druid Community Meetups
+layout: simple_page
+canonical: 'https://druid.apache.org/community/meetups'
+---
+
+## Druid Community Meetups
+
+Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to regular
+meetups in cities all over the world.
+
+All Apache Druid Meetups are covered the Apache [anti-harassment policy](https://www.apache.org/foundation/marks/events#anti-harassment) and speakers are expected to follow the guidelines below.
+
+### Speaker Guidelines
+
+**Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using, but only if they are a key part of your data architecture and Apache Druid. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.
+
+**Share Challenges:**  Feel good about sharing roadblocks you hit and how you worked around them – or ones you might still be struggling with – there may be a community member in the audience who can help you out! Engineering always involves trade-offs, so being realistic about them helps establish you as a credible advisor.
```

Comment:
We could add some example topics? Like

- Making contributions (docs, code)
- Tutorials - things people can do on their own, whether it's for real or just experimentation, complete with re-usable code.
- Coaching - like tips and tricks, guidance on how to use features, and lessons learned.